nixos/tests/all-tests.nix

```diff
@@ -710,6 +710,7 @@ in {
   victoriametrics = handleTest ./victoriametrics.nix {};
   vikunja = handleTest ./vikunja.nix {};
   virtualbox = handleTestOn ["x86_64-linux"] ./virtualbox.nix {};
+  vm = handleTest ./vm.nix {};
```

Comment:
https://nixos.org/manual/nixos/unstable/index.html#sec-calling-nixos-tests says that one should use `runTest`, but this file tends to use `handleTest` -- which is preferable?